src/pages/ThorChainLP/utils.ts

```diff
@@ -57,11 +57,13 @@ export const fromQuote = <T extends LpConfirmedDepositQuote | LpConfirmedWithdra
     return {
       assetId,
       depositType: type,
+      type,
     }
   }
 
   return {
     assetId,
     withdrawType: quote.withdrawSide,
+    type,
```

Comment:
quote withdrawSide is not the real initial side, we need the initial side to actually detect if for example we will be using thorchain to send a TX but to withdraw selecting the asset side (asset withdraw for a sym opportunity)